util/test/checkChplInstall

```diff
@@ -140,7 +140,7 @@ else
     echo "\$CHPL_LAUNCHER=${chpl_launcher} is not compatible with test script."
     echo "This does not necessarily indicate that your Chapel installation is incorrect."
     echo "See \$CHPL_HOME/doc/launcher.rst for information on manually launching Chapel programs."
-    exit 10
+    exit 0
```

Comment:
Yeah, we're going to have to do something special there anyways. I think we'll need to add support for chpl_launchcmd or something.

I forgot about this before, but under jenkins even slurm won't work because jenkins is "headless"(interactive slurm and pbs-aprun both require a TTY, which is why we have chpl_launchcmd in the first place.)

I think we'll either need to skip trying to run the test, or run it using chpl_lauchcmd.